src/Chainweb/Payload.hs

```diff
@@ -488,12 +483,22 @@ coinbaseOutputFromText t = either (throwM . TextFormatException . sshow) return
 
 -- | No-op coinbase payload
 --
-noCoinbase :: CommandResult a
-noCoinbase = CommandResult
-    (RequestKey pactInitialHash) Nothing
-    (PactResult (Right (PLiteral (LString "NO_COINBASE"))))
-    0 Nothing Nothing Nothing
-{-# NOINLINE noCoinbase #-}
+noCoinbaseOutput :: CoinbaseOutput
+noCoinbaseOutput = CoinbaseOutput $ encodeToByteString $ object
+    [ "gas" .= (0 :: Int)
+    , "result" .= object
+        [ "status" .= ("success" :: String)
+        , "data" .= ("NO_COINBASE" :: String)
+        ]
+    , "reqKey" .= ("DldRwCblQ7Loqy6wYJnaodHl30d3j3eH-qtFzfEv46g" :: String)
+        -- this is the unique hash value define in @Pact.Types.Hash.initialHash@
+    , "logs" .= Null
+    , "metaData" .= Null
+    , "continuation" .= Null
+    , "txId" .= Null
+    ]
+{-# NOINLINE noCoinbaseOutput #-}
+
```

Comment:
Why did this change?

Reply:
`Chainweb.Payload` was supposed to not depend on pact internals. `noCoinBaseOutput` is a constant that will never change. So it's fine to directly provide the content and avoiding  using constructors from pact types. The PR adds a unit test to ensure that both are in sync.

This simplifies the module graph a lot.

Reply:
Oh, and this value is only used in test-version of chainweb. It's not used in production.

Reply:
Yes, thank you. The pact dependency here should definitely go away.